### ISSUE TITLE
encoding of a file

### ISSUE THREAD

**Issue by Luuk34** [notepad-plus-plus/notepad-plus-plus#4202]
When opening a file, the correct encoding is shown in bottom right of screen ('UTF-8'), also the option 'Encode in UTF-8' is ticked under 'Encoding'
Changing this option to 'Encode in ANSI' changes the encoding of the displayed file, but does not change the bottom right notification which still says 'UTF-8', also the menu option under 'Encoding' shows a wrong value....

### Steps to Reproduce the Issue
1.  Open 'UTF-8' encoded file  (without BOM)
2.  change encoding to 'Encode in ANSI'

### Expected Behavior
Encoding tag in bottom right should change to 'ANSI'
Encoding in menu 'Encoding' should change to 'Encode in ANSI'

### Actual Behavior
- tag in bottom right does not (always) change   (only after changing tabs in npp, it seems to work)
- menu does not (always) change   (only after changing tabs in npp, it seems to work)

### Debug Information
Notepad++ v7.5.4   (32-bit)
Build time : Jan  1 2018 - 01:47:45
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : ComparePlugin.dll DSpellCheck.dll HexEditor.dll mimeTools.dll NppConverter.dll NppExport.dll NPPJSONViewer-win32.dll PluginManager.dll XMLTools.dll

**Comment by ArkadiuszMichalski** [comment]
No problem for me in Notepad++ 7.9 RC.
@sasumner maybe close and reopen if autor still have this problem and give more detail?

**Comment by Luuk34** [comment]
Sorry, i will refuse to comment on an issue from more then 2.5 years old...

Confirmation, or closing, of an issue should happen MUCH sooner,

**Comment by sasumner** [comment]
It could have been dealt with as another issue; there are a lot of open issues, and limited staff to process them.
